src/client/sandbox/node/index.js

```diff
@@ -46,7 +46,7 @@ export default class NodeSandbox extends SandboxBase {
     _processElement (el) {
         const processedContext = el[INTERNAL_PROPS.processedContext];
 
-        if (processedContext !== this.window) {
+        if (!domUtils.isShadowUIElement(el) && processedContext !== this.window) {
```

Comment:
minor enhancement: let's use a fast forward paradigm and remove one level of the brackets
```js
...
...
if (domUtils.isShadowUIElement(el) || processedContext === this.window)
   retrun;
...
```